sdk/src/main/java/io/customer/sdk/hooks/HooksManager.kt

```diff
@@ -0,0 +1,37 @@
+package io.customer.sdk.hooks
+
+import io.customer.sdk.util.Singleton
+
+interface HooksManager {
+    fun add(modules: HookModules, subscriber: ModuleHookProvider)
+    fun onHookUpdate(hook: ModuleHook)
+}
+
+enum class HookModules {
```

Comment:
```suggestion
enum class HookModule {
```

Since each enum case is a singular object, this name is less confusing. When I read `HookModules`, I expect to see something like a List and not a single enum case.

Reply:
Agreed, but did it to keep naming consistent with [iOS SDK](https://github.com/customerio/customerio-ios/blob/develop/Sources/Common/hooks/HooksManager.swift#L10)

Reply:
Ah! Good catch. I am just now realizing that the iOS SDK does that. I don't think that it should and I just changed that on the iOS codebase to match `HookModule`.